src/sync/treiber_stack.rs

```diff
@@ -77,6 +77,12 @@ impl<T> TreiberStack<T> {
     }
 }
 
+impl<T> Default for TreiberStack<T> {
+    fn default() -> Self {
+        TreiberStack { head: Atomic::null() }
```

Comment:
At least clippy recommends it the way it's implemented here: https://github.com/Manishearth/rust-clippy/blob/master/clippy_lints/src/new_without_default.rs#L34